src/main/java/seedu/fridgefriend/food/Fridge.java

```diff
@@ -0,0 +1,23 @@
+package seedu.fridgefriend.food;
+
+import java.util.ArrayList;
+
+public class Fridge {
+    private static ArrayList<Food> fridge = new ArrayList<>();
```

Comment:
Why is this class level?

Reply:
so we can statically access this without having to pass a Fridge object through methods